src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Parsing/StringsHelper.cs

```diff
@@ -119,5 +119,34 @@ public static string RemoveBracketsQuotesParenthesis(string value)
 
             return value;
         }
+
+        public static string ToSnakeCase(this string text)
```

Comment:
Need to add unit tests to get coverage for this PR up a bit.

Reply:
Good catch, done.